doc/source/cluster/configure-manage-dashboard.md

```diff
@@ -193,16 +193,21 @@ To view embedded time-series visualizations in Ray Dashboard, the following must
 1. The head node of the cluster is able to access Prometheus and Grafana
 2. The browser of the dashboard user is able to access Grafana. 
 
-Configure these settings using the `RAY_GRAFANA_HOST`, `RAY_PROMETHEUS_HOST`, and `RAY_GRAFANA_IFRAME_HOST` environment variables when you start the Ray Clusters.
+Configure these settings using the `RAY_GRAFANA_HOST`, `RAY_PROMETHEUS_HOST`, `RAY_PROMETHEUS_NAME`, and `RAY_GRAFANA_IFRAME_HOST` environment variables when you start the Ray Clusters.
 
 * Set `RAY_GRAFANA_HOST` to an address that the head node can use to access Grafana. Head node does health checks on Grafana on the backend.
 * Set `RAY_PROMETHEUS_HOST` to an address the head node can use to access Prometheus.
-* Set`RAY_GRAFANA_IFRAME_HOST` to an address that the user's browsers can use to access Grafana and embed visualizations. If `RAY_GRAFANA_IFRAME_HOST` not set, Ray Dashboard uses the value of `RAY_GRAFANA_HOST`.
+* Set `RAY_PROMETHEUS_NAME` is used when you select a different data source to use for the Grafana dashboard panles. Default is "Prometheus".
+* Set `RAY_GRAFANA_IFRAME_HOST` to an address that the user's browsers can use to access Grafana and embed visualizations. If `RAY_GRAFANA_IFRAME_HOST` not set, Ray Dashboard uses the value of `RAY_GRAFANA_HOST`.
 
 For example, if the IP of the head node is 55.66.77.88 and Grafana is hosted on port 3000. Set the value to `RAY_GRAFANA_HOST=55.66.77.88:3000`.
 
 If all the environment variables are set properly, you should see time-series metrics in {ref}`Ray Dashboard <observability-getting-started>`.
 
+:::{note}
+If your each ray cluster with one prometheus and share the Grafana, you should set the `RAY_PROMETHEUS_NAME` environment variable to different value and add these datasource in Grafana. Follow {ref}`these instructions <grafana>` to set up Grafana.
```

Comment:
```suggestion
If you set one Prometheus server for one Ray Cluster and use one central Grafana server for all Clusters, you should set the `RAY_PROMETHEUS_NAME` environment variable to different values for each Ray Cluster and add these datasource in Grafana. Follow {ref}`these instructions <grafana>` to set up Grafana.
```

Here is my suggestion. @angelinalg WDYT?

Reply:
@alanwguo @angelinalg WDYT?

Reply:
Not sure if should be "these datasources" or "the datasource". Whichever it is, just have the two words agree.

Reply:
I prefer to use "these datasources".